DataHub/UserInterface/02c_AttributeGroups.md

```diff
@@ -0,0 +1,232 @@
+  For detailed information about creating attribute groups or sub-groups, see [Create an attribute group](../Integration/03_ManageAttributeGroups.md#create-an-attribute-group) or [Create an attribute sub-group](../Integration/03_ManageAttributeGroups.md#create-an-attribute-sub-group).
+
+  > [Info] Attribute groups are organized in a tree structure with a maximum depth of two levels. That means that a maximum of one sub-group level can be added to an attribute group.
+
+
+### Add element
+
+*DataHub > Settings > Tab ATTRIBUTE GROUPS > Button Edit > Button Add*
+
+![Add element](../../Assets/Screenshots/DataHub/Settings/AttributeGroups/AddElement.png "[Add element]")
+
+> [Info] The *Add element* window for attribute groups and sub-groups is identical.
+
+- Language   
+  Click the drop-down list and select the system language in which the *Name (Language)* and *Description (Language)* fields are displayed. By default, the following options are available:
+  - **English (United States)**
+  - **Deutsch (Deutschland)**
+
+- *Name (Language)*   
+  Enter an attribute group name in the selected language.
+
+- *Key*   
+  Enter an attribute group key. The number of characters is limited to 190.
+
+- [CANCEL]   
+  Click this button to cancel adding an attribute group. The *Add element* window is closed.
+
+- [SAVE]   
+  Click this button to add the new attribute group. The *Add element* window is closed.
+
+  > [Info] You have to click the [SAVE] button in the *Edit attribute groups* view to save the added attribute group. Otherwise, all changes are rejected.
+
+
+
+### Edit attribute group data
+
+*DataHub > Settings > Tab ATTRIBUTE GROUPS > Button Edit > Hover over attribute group > Button Edit*
+
+![Edit attribute group data](../../Assets/Screenshots/DataHub/Settings/AttributeGroups/EditAttributeGroupData.png "[Edit attribute group data]")
+
+> [Info] The *Edit attribute group data* window for attribute groups and sub-groups is identical.
+
+
+- Language   
+  Click the drop-down list and select the system language in which the *Name (Language)* and *Description (Language)* fields are displayed. By default, the following options are available:
+  - **English (United States)**
+  - **Deutsch (Deutschland)**
+
+
+- *Name (Language)*   
+  Click the field to edit the attribute group name in the selected language.
+
+- *Key*   
+  Click the field to edit the attribute group key. The number of characters is limited to 190.
+
+- ![Delete](../../Assets/Icons/Trash01.png "[Delete]") (Delete)   
+  Click this button to remove the selected attribute group.
+
+  > [Info] You must click the [SAVE] button in the *Edit attribute groups* view to permanently delete the attribute group.
+
+  > [Warning] Be aware that as soon as you delete an attribute group, you also delete all assigned attribute sub-groups.
+
```

Comment:
[Caution] Loss of data
Deleting will permanently remove the selected data. The deletion cannot be undone and the deleted data cannot be restored.
Problems may occur due to unresolved dependencies.
Make sure you really want to delete the selected data.

Immer der Standardtext oder angepasst an der konkreten Situation?